tasks/publish.mk

```diff
@@ -33,6 +33,18 @@ $(PUB_SYSTEM_SYMBOLS): $(systemtarball)
 	$(hide) mkdir -p $(publish_dest)
 	tar --checkpoint=1000 --checkpoint-action=dot -czf $(publish_dest)/$@ $(PRODUCT_OUT)/symbols
 
+PUB_QEMU_SCRIPTS := qemu_scripts.tar.gz
+
+.PHONY: $(PUB_QEMU_SCRIPTS)
+$(PUB_QEMU_SCRIPTS): $(scriptstarball)
+	@echo "Publish scripts"
+	$(hide) mkdir -p $(publish_dest)
+	tar --checkpoint=1000 --checkpoint-action=dot -czf $(publish_dest)/$@ $(PRODUCT_OUT)/scripts
```

Comment:
can move all this codes to line 50, add lines break original system_symbols code.